qa/tasks/rgw.py

```diff
@@ -85,6 +85,29 @@ def start_rgw(ctx, config, clients):
                                                 kport=keystone_port),
                 ])
 
+        barbican_role = client_config.get('use-barbican-role', None)
+        if barbican_role is not None:
+            if not hasattr(ctx, 'barbican'):
+                raise ConfigError('rgw must run after the barbican task')
+
+            barbican_host, barbican_port = \
+                ctx.barbican.endpoints[barbican_role]
+            log.info("Use barbican url=%s:%s", barbican_host, barbican_port)
```

Comment:
The log lacks the protocol part, and thus shouldn't mention *URL*. I guess we might want to assemble `barbican_url` first and then use it here and for `rgw_cmd` as well.